mission-lotto/src/main/java/com/lotto/web/lotto/entity/LottoEntity.java

```diff
@@ -0,0 +1,42 @@
+package com.lotto.web.lotto.entity;
+
+import com.lotto.web.member.entity.Member;
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "LottoTicket")
+public class LottoEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "memberId")
+    private Member member;
+    @Column
+    private String lottoNumber;
```

Comment:
ㅎㅎ 이거 오해 있을까봐요 띄우는게 가독성면에서 좋아서 제안 드렸는데 이건 스타일이라 편하신대로 하셔도 됩니다!

```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "memberId")
    private Member member;
    
    @Column
    private String lottoNumber;
    
    @Column
    ...
    
```

Reply:
오.. 자바처럼 뭔가 컨벤션이 있는 줄 알았습니다! 저는 띄우는게 그래도 보기 편한거 같습니다..!